tests/strategies/factories.py

```diff
@@ -80,93 +77,74 @@ def coordinates_to_shaped_geometries(coordinates: Strategy[Coordinate]
 
 def coordinates_to_segments(coordinates: Strategy[Coordinate]
                             ) -> Strategy[Segment]:
-    return planar.segments(coordinates).map(Segment.from_raw)
+    return planar.segments(coordinates)
 
 
 def coordinates_to_multisegments(coordinates: Strategy[Coordinate]
                                  ) -> Strategy[Multisegment]:
     return planar.multisegments(coordinates,
-                                min_size=1).map(Multisegment.from_raw)
+                                min_size=2,
+                                max_size=MAX_LINEAR_SIZE)
 
 
 def coordinates_to_contours(coordinates: Strategy[Coordinate],
                             *,
-                            min_size: int = planar.TRIANGULAR_CONTOUR_SIZE,
-                            max_size: Optional[int] = None
+                            min_size: int = 3,
+                            max_size: int = MAX_LINEAR_SIZE
                             ) -> Strategy[Contour]:
-    return (planar.contours(coordinates,
-                            min_size=min_size,
-                            max_size=max_size)
-            .map(Contour.from_raw))
+    return planar.contours(coordinates,
+                           min_size=min_size,
+                           max_size=max_size)
 
 
 def coordinates_to_mixes(coordinates: Strategy[Coordinate]) -> Strategy[Mix]:
-    return coordinates_to_raw_mixes(coordinates).map(Mix.from_raw)
-
-
-def coordinates_to_raw_mixes(coordinates: Strategy[Coordinate]
-                             ) -> Strategy[RawMix]:
-    def coordinate_to_raw_empty(index: int, raw_mix: RawMix) -> RawMix:
-        return (raw_mix
-                if raw_mix[index]
-                else raw_mix[:index] + (RAW_EMPTY,) + raw_mix[index + 1:])
-
     return ((planar.mixes(coordinates,
                           min_multipoint_size=1,
                           min_multisegment_size=1)
-             .map(partial(coordinate_to_raw_empty, 2)))
-            | (planar.mixes(coordinates,
+             | planar.mixes(coordinates,
                             min_multipoint_size=1,
                             min_multipolygon_size=1)
-               .map(partial(coordinate_to_raw_empty, 1)))
-            | (planar.mixes(coordinates,
+             | planar.mixes(coordinates,
                             min_multisegment_size=1,
-                            min_multipolygon_size=1)
-               .map(partial(coordinate_to_raw_empty, 0))))
+                            min_multipolygon_size=1))
+            .map(pack(mix_from_packed_components)))
 
 
 def coordinates_to_polygons(coordinates: Strategy[Coordinate],
                             *,
-                            min_size: int = planar.TRIANGULAR_CONTOUR_SIZE,
+                            min_size: int = 3,
```

Comment:
What was wrong with the `TRIANGULAR_CONTOUR_SIZE`?

Reply:
It's no longer a part of hypothesis-geometry public API